### PR TITLE
EN-35571: Add ruby images with latest patch version

### PR DESCRIPTION
**Pull request by smooshy**

### CODE REVIEW

**Comment by smooshy** [comment]
I ran `bin/build` for each of the new images and it worked locally.

File: ruby/2.5.7/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+	&& { \
+		echo 'install: --no-document'; \
+		echo 'update: --no-document'; \
+	} >> /usr/local/etc/gemrc
+
+ENV RUBY_MAJOR 2.5
+ENV RUBY_VERSION 2.5.7
+ENV RUBY_DOWNLOAD_SHA256 201870e8f58957d542233fef588b1d76f7bf962fea44dcbd2237f4a5899a3f95
+ENV RUBYGEMS_VERSION 2.7.6
+ENV BUNDLER_VERSION 1.17
```
Comment (JoeNunnelley) [inline ruby/2.5.7/Dockerfile]: we should probably include logic to install both 1.x and 2.x versions of bundler in the files where we are installing ruby < 2.6. I believe 2.6+ versions come with bundler 2.0.x included.

File: ruby/2.6.5/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+	&& { \
+		echo 'install: --no-document'; \
+		echo 'update: --no-document'; \
+	} >> /usr/local/etc/gemrc
+
+ENV RUBY_MAJOR 2.6
+ENV RUBY_VERSION 2.6.5
+ENV RUBY_DOWNLOAD_SHA256 d5d6da717fd48524596f9b78ac5a2eeb9691753da5c06923a6c31190abe01a62
+ENV RUBYGEMS_VERSION 3.0.4
+ENV BUNDLER_VERSION 2.0.2
```
Comment (JoeNunnelley) [inline ruby/2.6.5/Dockerfile]: we may not need to do this for 2.6+ as I thought bundler was included in the distribution by default now.

File: ruby/2.5.7/Dockerfile (hunk 1 above)
Comment (smooshy) [reply ruby/2.5.7/Dockerfile]: I just installed `2.6.5` via rbenv and it looks like it installed `1.17.2`.

File: ruby/2.6.5/Dockerfile (hunk 2 above)
Comment (smooshy) [reply ruby/2.6.5/Dockerfile]: it looks like it's included, but it's installing `1.17.2` on my machine.

**Comment by smooshy** [comment]
@JoeNunnelley 👈 poke...

File: ruby/2.6.5/Dockerfile (hunk 2 above)
Comment (JoeNunnelley) [reply ruby/2.6.5/Dockerfile]: ok then lets stick with that.

**Pull request merged**